WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -24,8 +24,10 @@
   <ul>
     <% Team.order(:rank).each do |team| %>
       <li>
-        <strong><%= team.name %> (<%= team.acronym %>)</strong>: <%= format_team_members(team) %> <br />
-        <%= t("about.structure.#{team.friendly_id}.description") %>
+        <% if team.friendly_id != 'board' %>
```

Comment:
Maybe there's a better way of doing this.

Reply:
I would like us to store the string `'board'` somewhere else, maybe a `BOARD_FRIENDLY_ID` variable in the `Team` class? We could also do that with every other team, because we currently have those strings stashed in various calls to `team_member?` in `models/user.rb`

Also, I find it strange that this `if` statement is inside of the `<li>...</li>`. Does that mean we create an empty `<li>` for the Board Team? Maybe it would be clearer to put a "if board then next" right at the top of the `each` loop?

Reply:
You mean in case we might need to change the strings in the future?
That makes sense. I'll try to hunt them down.

About the loop, yeah, I was sure this was not the best way :P